crates/eframe/src/web/web_painter_wgpu.rs

```diff
@@ -193,7 +223,16 @@ impl WebPainter for WebPainterWgpu {
                             store: true,
                         },
                     })],
-                    depth_stencil_attachment: None,
+                    depth_stencil_attachment: self.depth_texture_view.as_ref().map(|view| {
+                        wgpu::RenderPassDepthStencilAttachment {
+                            view,
+                            depth_ops: Some(wgpu::Operations {
+                                load: wgpu::LoadOp::Clear(1.0),
+                                store: true,
```

Comment:
Quite common misconception :). You want to set `store` to false here since the texture is not read later anymore. This doesn't mean it is not written to as in no depth writes are happening, but rather that its contents can be discarded at the end of the renderpass. This means that if the gpu renders in a tile based fashion (the rule in mobile gpus, desktops are picking up as well!) it can optimize away writing to main gpu memory and keep the (potentially tiled) depth buffer only in on-chip memory!

```suggestion
                                store: false,
```